app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -19,6 +18,8 @@ import com.azavea.rf.utils.queryparams.QueryParametersCommon
 import com.azavea.rf.scene.SceneQueryParameterDirective
 
 
+case class BulkSceneOperation (action: String, sceneIds: Seq[UUID])
```

Comment:
We tend to add the case classes to the relevant datamodel object (in this case, it's /app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Bucket.scala)
Looks like the refactor moved it into a different directory.